.pre-commit-config.yaml

```diff
@@ -1,6 +1,6 @@
 repos:
   - repo: https://github.com/psf/black
-    rev: 21.4b0  # Replace by any tag/version: https://github.com/psf/black/tags
+    rev: 22.3.0  # Replace by any tag/version: https://github.com/psf/black/tags
```

Comment:
thanks for keeping the version pre-2023

Reply:
yeah this slipped in to this PR by mistake - I had to bump version to avoid some click <> black incompatibility that I did not get to the bottom of. but i guess we can leave it?

Reply:
yes `22.3.0` is totally fine, we can keep it in this PR.